root_VS2010/programs/C#/Frameworks/Infrastructure/Framework/Util/FxHttpSessionIndex.cs

```diff
@@ -59,6 +60,9 @@ public static class FxHttpSessionIndex
         /// <summary>ボタン履歴を保持する「キュー」を格納するSessionキー</summary>
         public const string BUTTON_HISTORY = "ButtonHistory";
 
+        /// <summary> Session key that contains the dummy value. </summary>
+        public const string DUMMY = "dummy";
```

Comment:
I think that this constant is unnecessary.

I think that this constant is used in the functions using ASP.NET MVC.
But, in root_VS2010, the functions using ASP.NET MVC do not exist.